src/core/github-api.js

```diff
@@ -20,32 +20,28 @@ export function githubRequestHeaders(conf) {
   return headers;
 }
 
-export async function getRateLimit(conf) {
-  const headers = githubRequestHeaders(conf);
-  const request = new Request(`https://api.github.com/rate_limit`, {
-    mode: "cors",
-    referrerPolicy: "no-referrer",
-    headers,
-  });
-  const responseJSON = await fetch(request).then(r => r.json());
-  return responseJSON.rate.remaining;
+export function checkLimitReached(response) {
+  if (response.ok) return false;
+  const { headers, status } = response;
+  if (status === 403 && headers.get("X-RateLimit-Remaining") === "0") {
+    if (typeof checkLimitReached.warned === "undefined") {
+      checkLimitReached.warned = true;
+      const msg = `You have run out of github requests. Some github assets will not show up.`;
+      pub("warning", msg);
+      window.pub = pub;
+    }
+    return true;
```

Comment:
```suggestion

```

Reply:
Slightly changed the code so it stays here.